policy/engine.go

```diff
@@ -495,11 +494,13 @@ func (e *Engine) query(ctx context.Context, input interface{}, query string) (ou
 
 			for _, v := range expressionValues {
 				switch val := v.(type) {
-
 				// Policies that only return a single string (e.g. deny[msg])
 				case string:
 					result := output.Result{
 						Message: val,
+						Metadata: map[string]interface{}{
```

Comment:
This change feels like it is separate from the SARIF change. Can you split out the PRs?

Reply:
I'll scope this out as a separate improvement